sle15/profiles/stig.profile

```diff
@@ -19,6 +19,7 @@ selections:
     - var_accounts_fail_delay=4
     - var_accounts_tmout=15_min
     - inactivity_timeout_value=15_minutes
+    - var_sudo_timestamp_timeout=1_minute
```

Comment:
```suggestion
    - var_sudo_timestamp_timeout=always_prompt
```

STIG recommends 0, even though positive values, I think the default one should be 0 and if people want to have different value here then they can customize it.